docs/victory-zoom-container/docs.md

```diff
@@ -0,0 +1,130 @@
+# VictoryZoomContainer
+
+`VictoryZoomContainer` provides pan and zoom behavior for any Victory component that works with an
+x, y axis. To add panning and zooming behavior, add `VictoryZoomContainer` as the
+`containerComponent` of your chart.
+
+```jsx
+<VictoryChart containerComponent={<VictoryZoomContainer/>}>
+  <VictoryLine data={data} />
+  <VictoryBar data={moreData}/>
+</VictoryChart>
+```
+
+## Props
+
+`VictoryZoomContainer` uses a superset of props used by [VictoryContainer].
+
+
+### zoomDomain
+
+The optional `zoomDomain` prop describes the zoomed and panned state. This prop is an object that
+specifies separate arrays for x and y. Each array is a tuple that describes the minimum and maximum
+values to render. If this prop is not provided initially, the chart will render without an initial
+zoom. Updates to `zoomDomain` will trigger a re-render of the chart with the new domain.
+
+*examples:* `zoomDomain={{x: [0, 100]}`
```

Comment:
If you set zoomDomain statically like this it would basically break zooming right? Meaning it would aways be zoomed into this part?